### ISSUE TITLE
confused about softmax(v_length)

### ISSUE THREAD

**Issue by BenchengY** [naturomics/CapsNet-Tensorflow#36]
I am confused that why you use  the softmax to v_length [here](https://github.com/naturomics/CapsNet-Tensorflow/blob/5e3e64e6d09557f92f9cd5a6f15201d12fe93b59/capsNet.py#L71). since i have not found this operate in hitton's paper and Figure 1? 
In contrast, it seems that the capsnet allows a muti-label classification which means that it is not necessary to use the softmax to v_length, according to  Section 3 of hitton's paper (To allow for multiple digits, we use a separate margin),
